pylint/test/functional/used_before_assignment_issue1081.py

```diff
@@ -0,0 +1,32 @@
+# pylint: disable=missing-docstring,invalid-name,redefined-outer-name
```

Comment:
Just to make sure we don't break errors about shadowing with this change, can you not disable `redefined-outer-name` and ensure Line 15, 21, and 30 still generate that error?